app/models/lesson.rb

```diff
@@ -10,6 +10,6 @@
 #
 
 class Lesson < ApplicationRecord
-  validates :title, presence: true, length: { maximum: 50 }
+  validates :title, presence: true, allow_blank: false, length: { maximum: 50 }
```

Comment:
Despite what was said in last video, after some research `allow_blank` is not needed (see our slack discussion https://thehackingproject.slack.com/archives/CBA9A9TC4/p1531828100000104)